infrastructure/lib/constructs/compliant-resources.ts

```diff
@@ -24,7 +24,8 @@ export class SecureBucket extends cdk.Construct {
       // secure defaults that cannot be overridden
       publicReadAccess: false,
       blockPublicAccess: s3.BlockPublicAccess.BLOCK_ALL,
-      autoDeleteObjects: false,
+      // NOTE: removed for dev purposes only, once envs are differentiated this will be fixed
```

Comment:
Can you change this to phrasing that includes `TODO` so that it's more searchable?
```suggestion
      // TODO: Prevent auto-delete when we properly handle differences between dev and
      // higher environments
```

or something like that

Reply:
Yeah, this is the second time today. I need to just go with my gut (I had `TODO` originally).